cdn.tf

```diff
@@ -131,6 +139,48 @@ resource "azurerm_cdn_frontdoor_rule" "redirect" {
   }
 }
 
+resource "azurerm_cdn_frontdoor_rule" "url_path_redirect" {
+  for_each = local.waf_application == "CDN" ? {
+    for index, host_redirect in local.cdn_url_path_redirects : index => {
+      redirect_type        = host_redirect.redirect_type,
+      redirect_protocol    = lookup(host_redirect, "redirect_protocol", null),
+      destination_path     = lookup(host_redirect, "destination_path", null),
+      destination_hostname = lookup(host_redirect, "destination_hostname", null),
+      destination_fragment = lookup(host_redirect, "destination_fragment", null),
+      query_string         = lookup(host_redirect, "query_string", null),
+      operator             = host_redirect.operator,
+      match_values         = lookup(host_redirect, "match_values", null),
+      transforms           = lookup(host_redirect, "transforms", null)
+    }
+  } : {}
+
+  depends_on = [azurerm_cdn_frontdoor_origin_group.waf, azurerm_cdn_frontdoor_origin.waf]
+
+  name                      = "urlredirect${each.key}"
```

Comment:
Is it worth changing this to be something more descriptive, e.g. based off one of the redirect values?

Reply:
The only non-nullable ones are operator & request type, I could add these but I'm not sure they'll add much over the standard `rule{x}`